### PR TITLE
jstat documentation

### PR DESCRIPTION
**Pull request by JasonFengJ9**
**jstat documentation**

1. Created  `tool_jstat.md`;
2. Added an entry in `mkdocs.yml`;
3. Updated 0.18.0 "What's New" topic;
4. Updated `tool_migration.md`.

Reviewer: @SueChaplain 

Signed-off-by: Jason Feng <fengj@ca.ibm.com>

### CODE REVIEW

File: docs/tool_jstat.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+# Java statistics monitoring (`jstat`) tool
+
+Use the `jstat` tool to obtain Java Virtual Machine (JVM) statistics. The tool is similar to the HotSpot tool of the same name; the OpenJ9 version of `jstat` is an independent implementation, added for compatibility.
+
+The command syntax is as follows:
+
+    jstat [<option>] [<vmid>]
+
+Where `vmid` is the Attach API virtual machine identifier for the Java process. This ID is typically the same as
```
Comment (pmhayward) [inline docs/tool_jstat.md]: Minor thing, but better with a small 'w' on 'where...' as it's not a new sentence.

File: docs/tool_jstat.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+the operating system *process ID*, unless you specified the `-Dcom.ibm.tools.attach.id` system property when you started the process.
```
Comment (pmhayward) [inline docs/tool_jstat.md]: Would be good to cross reference:  [`-Dcom.ibm.tools.attach.id`](dcomibmtoolsattachid.md)

File: docs/tool_migration.md [hunk 3]
```diff
@@ -71,5 +71,12 @@ Displays information about Java stack traces and thread information for processe
 
 For more information, see [`jstack`](tool_jstack.md).
 
+#### Java statistics monitoring (`jstat`) tool
+
+Displays information about Java statistics for processes. The main differences from the HotSpot `jstat` tool are as follows:
```
Comment (pmhayward) [inline docs/tool_migration.md]: Only one difference(?) so make singular:  "The main difference from the HotSpot `jstat` tool is as follows:..."
(Or just remove the (one) bullet point and merge into the sentence.)

File: docs/tool_jstat.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+VMID is shown in `jps` or other Attach API-based tools.
+
+On its own, `jstat` prints help information. The values for `<options>` are as follows:
```
Comment (pmhayward) [inline docs/tool_jstat.md]: Is there a reason why the two parameters are not listed in the same order as in the syntax?

File: docs/tool_jstat.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+Use the `jstat` tool to obtain Java Virtual Machine (JVM) statistics. The tool is similar to the HotSpot tool of the same name; the OpenJ9 version of `jstat` is an independent implementation, added for compatibility.
+
+The command syntax is as follows:
+
+    jstat [<option>] [<vmid>]
+
+Where `vmid` is the Attach API virtual machine identifier for the Java process. This ID is typically the same as
+the operating system *process ID*, unless you specified the `-Dcom.ibm.tools.attach.id` system property when you started the process.
+
+VMID is shown in `jps` or other Attach API-based tools.
```
Comment (pmhayward) [inline docs/tool_jstat.md]: Again, a cross reference would be good: ... in the [Java process status (jps) tool](tool_jps.md) or other...

File: docs/tool_jstat.md (hunk 1 above)
Comment (JasonFengJ9) [reply docs/tool_jstat.md]: fixed.

File: docs/tool_jstat.md (hunk 2 above)
Comment (JasonFengJ9) [reply docs/tool_jstat.md]: fixed.

File: docs/tool_migration.md (hunk 3 above)
Comment (JasonFengJ9) [reply docs/tool_migration.md]: Currently this tool is really a start point for migration purpose, extra features can be added as per user requests.
Updated the sentence.

File: docs/tool_jstat.md (hunk 5 above)
Comment (JasonFengJ9) [reply docs/tool_jstat.md]: fixed.

File: docs/tool_jstat.md (hunk 4 above)
Comment (JasonFengJ9) [reply docs/tool_jstat.md]: It meant `<option>` in the syntax above.

Comment (JasonFengJ9) [reply docs/tool_jstat.md]: Please have another look.

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by JasonFengJ9** [comment]
What's the purpose of `Merge branch 'master' into jstat-doc`?

**Comment by JasonFengJ9** [comment]
@pmhayward any further comment about this PR?

**Comment by JasonFengJ9** [comment]
@SueChaplain what's the next action for this PR?

**Pull request merged**